src/components/DimensionsPanel/__tests__/__snapshots__/DimensionsPanel.spec.js.snap

```diff
@@ -0,0 +1,164 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DimensionsPanel matches the snapshot 1`] = `
+<div>
+  <div
+    style="height: 100%; display: flex; flex-direction: column; background-color: rgb(255, 255, 255); padding: 8px; overflow: hidden;"
+  >
+    <div
+      class="container"
+    >
+      <div
+        class="jsx-1141087211 "
```

Comment:
Imho we should avoid snapshot testing when ui components end up in the snapshot because these classes will sometimes get new ids. An alternative would be to mock the ui components.

Reply:
Refactored to not rely on the snapshot.